tests/rados/test_osd_ecpool_inconsistency_scenario.py

```diff
@@ -125,26 +140,18 @@ def run(ceph_cluster, **kw):
                 f"Scrub repaired the {no_of_objects - obj_count} inconsistent objects"
             )
             return 1
-        obj_count = get_inconsistent_count(scrub_object, pg_id, rados_obj, "deep-scrub")
-        if obj_count == -1:
-            log.error(f"The deep-scrub not initiated on the pg-{pg_id}")
-            rados_obj.log_cluster_health()
-            return 1
-        if obj_count != no_of_objects:
-            log.error(
-                f"deep-scrub repaired the {no_of_objects - obj_count} inconsistent objects"
-            )
-            rados_obj.log_cluster_health()
-            return 1
         log.info(
-            "Verification of Test scenario1- inconsistent objects greater than the "
-            "osd_scrub_auto_repair_num_errors count completed "
+            "Test scenario1 completed:SCRUB-Inconsistent objects greater than the "
+            "osd_scrub_auto_repair_num_errors count"
         )
-        # Case2: The inconsistent object count is less than osd_scrub_auto_repair_num_errors count
+        # Case2: The inconsistent object count is less than osd_scrub_auto_repair_num_errors count and
+        # osd_scrub_auto_repair is false
         auto_repair_param_value = no_of_objects + 1
         log.info(
-            "Test scenario2: inconsistent objects less than the osd_scrub_auto_repair_num_errors count "
+            "Test scenario2: SCRUB-Inconsistent objects less than the osd_scrub_auto_repair_num_errors count and "
```

Comment:
Can you add the expected behaviour for each case? 

like : 
operation : scrub
Expectation : No repairs to be made.
Inconsistent objects before and after  test workflow : {}